peepholelib/models/model_wrap.py

```diff
@@ -5,6 +5,40 @@
 # torch stuff
 import torch
 
+import torch.nn as nn
+from collections import OrderedDict
+from torch import Tensor
+
+means = {
+        'CIFAR10': torch.tensor([0.424, 0.415, 0.384]),
+        'CIFAR100': torch.tensor([0.438, 0.418, 0.377]),
+        'ImageNet': torch.tensor([0.485, 0.456, 0.406]),
+        'SVHN': torch.tensor([0.438, 0.444, 0.473])
+        }
+
+stds = {
+        'CIFAR10': torch.tensor([0.283, 0.278, 0.284]),
+        'CIFAR100': torch.tensor([0.300, 0.287, 0.294]),
+        'ImageNet': torch.tensor([0.229, 0.224, 0.225]),
+        'SVHN': torch.tensor([0.198, 0.201, 0.197]),
+        }
+
+'''Inspired by https://github.com/RobustBench/robustbench/blob/master/robustbench/model_zoo/architectures/utils_architectures.py'''
+
+class ImageNormalizer(nn.Module):
+
+    def __init__(self, mean, std):
+        super(ImageNormalizer, self).__init__()
+
+        self.register_buffer('mean', mean.view(1, 3, 1, 1))
```

Comment:
These `view()` calls should be unnecessary; the means and stds should be passed in the correct format. This will make it easier to generalize this module to an "input normalizer"